backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/VirtualQueueRepository.java

```diff
@@ -16,10 +16,13 @@
 
 package com.google.gpay.virtualqueue.backendservice.repository;
 
+import java.util.Map;
 import java.util.UUID;
 import com.google.gpay.virtualqueue.backendservice.proto.CreateShopRequest;
 
 public interface VirtualQueueRepository {
 
-	public UUID createShop(CreateShopRequest shop);
+    public UUID createShop(CreateShopRequest shop);
+
```

Comment:
Nit. Can we remove this newline from here or is it required?

Reply:
I guess we should have a line between function declaration. Not sure tough. Please confirm.

Reply:
yes read that there should be new line after function declarations